### PR TITLE
test/e2e: move funcs from test/e2e/pod to other folders

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

move funcs from test/e2e/pod to 
```
test/e2e/framework/util.go
test/e2e/scheduling/framework.go
test/e2e/scheduling/predicates.go
test/e2e/storage/persistent_volumes.go
test/e2e/storage/utils/create.go
```

**Which issue(s) this PR fixes**:

Ref #86052

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/storage/utils/create.go [hunk 1]
```diff
@@ -624,3 +624,17 @@ func PrettyPrint(item interface{}) string {
 	}
 	return fmt.Sprintf("%+v", item)
 }
+
+// PatchContainerImages replaces the specified Container Registry with a custom
```
Comment (SataQiu) [inline test/e2e/storage/utils/create.go]:
```suggestion
// patchContainerImages replaces the specified Container Registry with a custom
```

File: test/e2e/storage/persistent_volumes.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ package storage
 
 import (
 	"fmt"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
```
Comment (SataQiu) [inline test/e2e/storage/persistent_volumes.go]: `e2elog` will be removed later.
Please use `"k8s.io/kubernetes/test/e2e/framework"` instead.
`framework.Logf...`

**Review by SataQiu — COMMENTED** [review]
Small nits. Others lgtm.

**Comment by hwdef** [comment]
@SataQiu 
Thanks for  your review, I fixed them.

**Review by oomichi — COMMENTED** [review]
/lgtm

**Review by SataQiu — COMMENTED** [review]
Thanks 
/lgtm
/approve

**Pull request merged**